htdocs/comm/index.php

```diff
@@ -190,10 +191,13 @@
 				$i++;
 				$total += $obj->total_ht;
 			}
-			if ($total>0)
+			if ($num > $nbofloop)
 			{
-
-				print '<tr class="liste_total"><td>'.$langs->trans("Total").'</td><td colspan="2" class="right">'.price($total)."</td></tr>";
+				print '<tr class="liste_total"><td colspan="3" align="right">'.$langs->trans("XMoreLines", ($num - $nbofloop))."</td></tr>";
```

Comment:
@altatof Use html5 class="center" instead of align="right"

Reply:
are you sure that class="center" will align the cell on the right ?